pants.toml

```
@@ -12,20 +12,31 @@ backend_packages.add = [
   "pants.backend.python.typecheck.mypy",
 ]
 
+use_deprecated_directory_cli_args_semantics = true
```

Sorry, I gave you bad advice

```suggestion
use_deprecated_directory_cli_args_semantics = false
```